boogiestats/boogie_api/test/test_api.py

```diff
@@ -76,10 +82,14 @@ def test_player_scores_given_groovestats_ranked_song(client, gs_api_key, request
     }
     requests_mock.get(GROOVESTATS_ENDPOINT + "/player-scores.php", text=json.dumps(ranked_song))
     kwargs = {
-        f"HTTP_X_Api_Key_Player_{player_index}": gs_api_key,
+        f"HTTP_x_api_key_player_{player_index}": gs_api_key,
```

Comment:
normalize other instances of it as well. Can it be snake_case or is `HTTP_` prefix meaningful?

Reply:
it turns out that it has to be `HTTP_`